### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by NovusEdge**

### CODE REVIEW

File: system-diagram.jpg
Comment (0mlml) [inline system-diagram.jpg:1]: this should be in `.github/img/`

**Review by 0mlml — APPROVED** [review]
just move the image otherwise good

File: README.md [hunk 1]
```diff
@@ -1 +1,81 @@
...
+
+4. **User Feedback:**
+   - The device's user interface provides live updates on:
+     - **Sensor Readings:** Displays real-time temperature, humidity, and movement data.
+     - **Placement Status:** Indicates whether the probe is "In the mouth" or "Not in the mouth."
+
+5. **Connectivity:**
+   - BLE, LoRa, and AWS Sidewalk support ensure seamless data transfer and integration with external systems if needed.
+
+This combination of sensors, intelligent algorithms, and responsive design makes the IMP a robust and innovative solution to its uniquely playful challenge.
```
Comment (0mlml) [inline README.md]: "...innovative solution to its uniquely playful challenge." 
??

**Review by 0mlml — CHANGES_REQUESTED** [review]
just move the image

File: README.md (hunk 1 above)
Comment (NovusEdge) [reply README.md]: Stupid AI rephrasing shit

**Comment by NovusEdge** [comment]
Changes made, lemme know if it's all g now

**Review by 0mlml — APPROVED** [review]

**Pull request merged**